integration-tests/gatsby-source-wordpress/.env.test

```diff
@@ -1,12 +1,11 @@
-WPGRAPHQL_VERSION=v1.3.7
-WPGATSBY_VERSION=v1.0.10
-WPGRAPHQL_ACF_VERSION=v0.3.1
-WPGRAPHQL_CPT_UI_VERSION=v1.1
-CPT_UI_VERSION=1.8.1
+WPGRAPHQL_VERSION=v1.8.0
+WPGATSBY_VERSION=v2.3.2
+WPGRAPHQL_ACF_VERSION=v0.5.3
+CPT_UI_VERSION=1.11.2
 ACF_VERSION=5.8.7
```

Comment:
I didn't update ACF because there's no oss mirror of the pro version with a version higher than this